src/pages/guides/using/asset-events/asset-events-properties.md

```diff
@@ -0,0 +1,151 @@
+# Asset Events Properties
+
+<br/>
```

Comment:
Can you remove this line break as well @sangeetha5491 please?

Reply:
done